site/content/en/docs/drivers/docker.md

```diff
@@ -35,6 +35,14 @@ The Docker driver allows you to install Kubernetes into an existing Docker insta
 
 [comment]: <> (this title is used in the docs links, don't change)
 
+### Prune docker to avoid performance downgrade in minikube
+ the following command will prune all on unused/stopped containers, volumes on your docker.
```

Comment:
Provide contextual documentation on why this improves performance.

Also, remove the leading whitespace and capitalize "the".